README.md

```diff
@@ -1 +1,17 @@
 # das-grafana
+
+## User Management
+
+User management is handled by a combination of Auth0 and GitHub, this integration prevents login using the admin account from a browser.  After deploying a new instance of das-grafana you will need to add users to the admin roles by making API calls using the admin account.  The admin credentials are stored in a Kubernetes secret.  Execute the following PowerShell to add a user to an admin role.  First login with your user via the Auth0 GitHub flow then execute the following PowerShell to make your user an admin:
```

Comment:
I can though the source of truth is always going to be the Azure DevOps variable which isn't readable.  As the secret referred to above derives directly from that I think it's better to refer to that in the documentation rather than a manually created copy in KeyVault